client-src/README.md

```diff
@@ -0,0 +1,275 @@
+# Logistics Wizard Demo
+
+[![Build Status](https://travis-ci.org/davezuko/react-redux-starter-kit.svg?branch=master)](https://travis-ci.org/davezuko/react-redux-starter-kit?branch=master)
```

Comment:
Yea I have been updating the readme as I go, just forgot about that section. I have a lot more changes locally I'll push up on next merge. Thanks for catching this one.